lib/rules/web/admin_console/4.4/projects.xyaml

```diff
@@ -1,6 +1,6 @@
 #---check project related action
 create_project:
-  action: click_create_button
+  action: click_create_project_button
```

Comment:
if we use `click_create_button` then another Create button will be clicked, see
<img width="1370" alt="Screen Shot 2020-04-01 at 6 39 00 PM" src="https://user-images.githubusercontent.com/12692381/78128220-27a7d800-7448-11ea-9301-38e1a20adcfe.png">